modules/nf-core/wfmash/tests/main.nf.test.snap

```diff
@@ -1,33 +1,10 @@
 {
-    "homo_sapiens - pangenome - pangenome_fa_bgzip": {
-        "content": [
-            {
-                "0": [
-                    [
-                        {
-                            "id": "test",
-                            "single_end": false
-                        },
-                        "test.paf:md5,aa6629548e32b2d036c0767c9086a6dd"
-                    ]
-                ],
-                "1": [
-                    "versions.yml:md5,0da45b7a6424a967e2f39225fb020140"
-                ],
-                "paf": [
-                    [
-                        {
-                            "id": "test",
-                            "single_end": false
-                        },
-                        "test.paf:md5,aa6629548e32b2d036c0767c9086a6dd"
-                    ]
-                ],
-                "versions": [
-                    "versions.yml:md5,0da45b7a6424a967e2f39225fb020140"
-                ]
-            }
-        ],
-        "timestamp": "2024-01-25T15:12:21.284828212"
+    "version": {
+        "content": null,
```

Comment:
that's fishy. there should be a versions.yml with a md5sum. Can you check if the version output command in the main.nf is correct?

Reply:
I never touched that.

```
$(echo $(wfmash --version 2>&1) | cut -f 1 -d '-' | cut -f 2 -d 'v')
bash: 0.13.0: command not found
```

This looks good to me. Any ideas what the problem is. I feel lost.

Reply:
Re-running the snapshot generation does not change anything.

```
nf-core modules test --update

                                          ,--./,-.
          ___     __   __   __   ___     /,-._.--~\
    |\ | |__  __ /  ` /  \ |__) |__         }  {
    | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                          `._,._,'

    nf-core/tools version 2.13.1 - https://nf-co.re


? Tool name: wfmash
? Choose container software to run the test with Docker
INFO     Generating nf-test snapshot                                                                                                                                                
 ╭───────────────────────────────────────────────────────────────────────────────── nf-test output ─────────────────────────────────────────────────────────────────────────────────╮
│                                                                                                                                                                                  │
│ 🚀 nf-test 0.8.4                                                                                                                                                                 │
│ https://code.askimed.com/nf-test                                                                                                                                                 │
│ (c) 2021 - 2024 Lukas Forer and Sebastian Schoenherr                                                                                                                             │
│                                                                                                                                                                                  │
│ Found 1 files in test directory.                                                                                                                                                 │
│ Warning: every snapshot that fails during this test run is re-record.                                                                                                            │
│                                                                                                                                                                                  │
│ Test Process WFMASH                                                                                                                                                              │
│                                                                                                                                                                                  │
│   Test [8c0ee97a] 'homo_sapiens - pangenome - pangenome_fa_bgzip' PASSED (9.589s)                                                                                                │
│                                                                                                                                                                                  │
│                                                                                                                                                                                  │
│ SUCCESS: Executed 1 tests in 9.592s                                                                                                                                              │
│                                                                                                                                                                                  │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
INFO     Generating nf-test snapshot again to check stability                                                                                                                       
╭───────────────────────────────────────────────────────────────────────────────── nf-test output ─────────────────────────────────────────────────────────────────────────────────╮
│                                                                                                                                                                                  │
│ 🚀 nf-test 0.8.4                                                                                                                                                                 │
│ https://code.askimed.com/nf-test                                                                                                                                                 │
│ (c) 2021 - 2024 Lukas Forer and Sebastian Schoenherr                                                                                                                             │
│                                                                                                                                                                                  │
│ Found 1 files in test directory.                                                                                                                                                 │
│                                                                                                                                                                                  │
│ Test Process WFMASH                                                                                                                                                              │
│                                                                                                                                                                                  │
│   Test [8c0ee97a] 'homo_sapiens - pangenome - pangenome_fa_bgzip' PASSED (8.709s)                                                                                                │
│                                                                                                                                                                                  │
│                                                                                                                                                                                  │
│ SUCCESS: Executed 1 tests in 8.712s                                                                                                                                              │
│                                                                                                                                                                                  │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
INFO     All tests passed!
```

Reply:
In the old snapshot, there still was such a yml. 
I have nf-core v2.13.1 and nf-test v0.8.4.

Reply:
are you sure about that the command works. tried to install it via conda on gitpod and got:
```console
wfmash --version
Flag could not be matched: version
  wfmash [target] [queries...] {OPTIONS}

    wfmash: base-accurate alignments using mashmap2 and the wavefront algorithm

  OPTIONS:

Illegal instruction
```

Reply:
@mashehu For me this worked:

```
mamba create --name wfmash0130 wfmash=0.13.0
mamba activate wfmash0130
which wfmash
/home/heumos/miniconda3/envs/wfmash0130/bin/wfmash
wfmash --version
v0.13.0-0-gd7b6960
```
Let me try the docker image.

My log above is absolutely fine. The command that I executed returns `0.13.0` to bash. And bash wants to execute `0.13.0` which results in `command not found`.

Reply:
Maybe @AndreaGuarracino has an idea? What could cause the illegal instruction above? Thanks!

Reply:
It could be that in the wfmash recipe you need to actually add extra flags like in the odgi recipe https://github.com/bioconda/bioconda-recipes/blob/86e7488a791a450d56e5d5c47f977c1b88f76111/recipes/odgi/build.sh#L10  for WFA2-lib and not just replace the flags of the release build type as you currently do in https://github.com/bioconda/bioconda-recipes/blob/86e7488a791a450d56e5d5c47f977c1b88f76111/recipes/wfmash/build.sh#L10.

Reply:
@AndreaGuarracino the actual problem for this PR originates from somewhere else, ignore the above